### ISSUE TITLE
simple test case error for sphere

### ISSUE THREAD

**Issue by tague-carlyon** [firemodels/fds#11896]
[geomtest.txt](https://github.com/firemodels/fds/files/11727957/geomtest.txt)

When I attempt to run the simulation, I get an access violation.

**Comment by marcosvanella** [comment]
Thanks for reporting. I'll take a look.

**Comment by marcosvanella** [comment]
Hi Tague, this is an issue stemming from a very fine grid in your sphere (come FDS cells contain 1200+ triangles of the sphere). 
I'll work on it. Also, I recommend you to make the size of the triangles from 1/10 of the FDS cell size to as large as needed.

**Comment by DorkishDude** [comment]
With a smaller mesh size, the simulation does not refuse to run anymore. Thank you.

**Comment by marcosvanella** [comment]
Ok Tague, this should be fixed with PR #11902. Tomorrow you can install tonights test bundle and try the case with the new executable.

**Comment by DorkishDude** [comment]
[serverrack45simp.txt](https://github.com/firemodels/fds/files/11763248/serverrack45simp.txt)

Okay, I tested a similar sphere case, and it seems to work. Unfortunately, I did not archive that specific case, and I have learned my lesson. The geomtest.txt file is corrupt when Pyrosim attempts to load it.

That being said, there is another case that is also having a memory access issue posted above.
